### PR TITLE
Adding permissions in backplane for CEE to support ACS:CS

### PR DESCRIPTION
**Pull request by pat-cremin**
### What type of PR is this?
Enabling access to ACS for CEE for GA

### What this PR does / why we need it?
Enabling access to ACS for CEE for GA
### Which Jira/Github issue(s) this PR fixes?

_Fixes #_
[MTSRE-1881](https://issues.redhat.com/browse/MTSRE-1881)
### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

**Comment by jewzaam** [comment]
/hold

no jira link
no documentation updated

**Comment by pat-cremin** [comment]
@jewzaam  I had Jira creation issues earlier. I have created and linked the Jira to the PR now.

**Comment by pat-cremin** [comment]
re: `no documentation updated`

I see no documentation within the repo other than the general structure in the `deploy/backplane` README.md. Did I overlook something specific that you could point me to?

**Comment by jewzaam** [comment]
> re: `no documentation updated`
> 
> I see no documentation within the repo other than the general structure in the `deploy/backplane` README.md. Did I overlook something specific that you could point me to?

https://github.com/openshift/managed-cluster-config/tree/master/docs/backplane

File: docs/backplane/requirements/cee/40-backplane-cee.md
Comment (jewzaam) [inline docs/backplane/requirements/cee/40-backplane-cee.md:18]:
```suggestion
## backplane-readers-cluster: `cluster`

Existing role, permission to view select resources at cluster scope.

## backplane-cee-readers-cluster: `cluster`

* view machine configurations
* view machines
* view api request counts
* view egress ips

## openshift-backplane-cee: `(^kube$|^kube-.*|^openshift$|^openshift-.*|^default$|^redhat-.*|^rhacs$|^rhacs-*)`

* view resources in namespaces using `dedicated-readers` role
```

**Review by jewzaam — CHANGES_REQUESTED** [review]

**Comment by jewzaam** [comment]
Suggested updates to the cee backplane requirements doc.  Will approve and ask someone lgtm and remove hold when doc updates are good enough.

/approve

**Review by jewzaam — APPROVED** [review]
/hold cancel

**Pull request merged**
